packages/lesswrong/unitTests/schemacheck.test.ts

```diff
@@ -15,7 +15,7 @@ describe('Schema check', () => {
       forumType: "EAForum",
       silent: true,
     });
-  }, 60000);
+  }, 120000);
```

Comment:
This change has made `yarn makemigrations` quite a bit slower - it's still not too bad locally but it was just about hitting the 1 minute timeout in github actions where the runners are pretty underpowered. I don't think it's worth fixing in this PR but it'd probably be relatively straight forward to improve the performance in the future with a bit of profiling as it shouldn't take as long as it does - it could just be something simple like needing to use a different hash function.

Reply:
Not sure how helpful it'd be but it looks like there's a lot of string concatenation which could be replaced by a `.join()` on a stringbuilder array; that's the kind of thing that might have gotten worse if we've both made the final output longer and also added a bunch of additional iterations.  (Maybe node has optimized string concatenation enough that it no longer matters?)